addons/mail/static/tests/new/discuss/sidebar_tests.js

```diff
@@ -248,3 +248,81 @@ QUnit.test("sidebar: basic chat rendering", async function (assert) {
     assert.containsOnce($chat, ".o-mail-commands div[title='Unpin Conversation']");
     assert.containsNone($chat, ".badge");
 });
+
+QUnit.test("sidebar: show pinned channel", async function (assert) {
+    const pyEnv = await startServer();
+    pyEnv["mail.channel"].create({ name: "General" });
+    const { openDiscuss } = await start();
+    await openDiscuss();
+    assert.containsOnce(target, ".o-mail-category-item:contains(General)");
+});
+
+QUnit.test("sidebar: open pinned channel", async function (assert) {
+    const pyEnv = await startServer();
+    pyEnv["mail.channel"].create({ name: "General" });
+    const { openDiscuss } = await start();
+    await openDiscuss();
+    await click(".o-mail-category-item:contains(General)");
+    assert.strictEqual($(target).find(".o-mail-discuss-thread-name").val(), "General");
```

Comment:
Not sure about using JQuery, it does not look like something that can't be done with vanilla JS. I'd rather replacing JQuery calls in this file.